Jenkinsfile

```diff
@@ -1 +1,13 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
\ No newline at end of file
+buildPlugin(configurations: [
+          // Sanity on the minimum required Jenkins version
+         [ platform: "linux", jdk: "8", jenkins: '2.190.3' ],
```

Comment:
This does not make sense. https://github.com/jenkinsci/electricflow-plugin/blob/2e6b4a58cc763d722780359cbca40ff0a33ec55e/pom.xml#L17 already, do this is the same as

```groovy
[ platform: "linux", jdk: "8" ]
```

for now at least. At best it is redundant; at worst it will break if the POM version is increased.

Reply:
I somehow missed fact that this will override the one specified in pom.xml. I will remove the line in a separate PR